Tests/Filters/SiteLogoEnhancedTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+namespace Bcgov\Theme\Block\Tests\Filters;
+
+use Bcgov\Theme\Block\Tests\CommonTestCase;
+use Bcgov\Theme\Block\Filters\SiteLogoEnhanced;
+
+/**
+ * SiteLogoEnhanced test.
+ */
+class SiteLogoEnhancedTest extends CommonTestCase {
+    /**
+     * The SiteLogoEnhanced instance.
+     *
+     * @var SiteLogoEnhanced
+     */
+    protected $logo;
+
+    /**
+     * Used for setup of each test.
+     *
+     * @inheritDoc
+     */
+    public function setUp() :void {
+        parent::setUp();
+        $this->logo = new SiteLogoEnhanced();
+    }
+
+
+    /**
+     * Test the site logo with no block contents, should return empty string.
+     *
+     * @return void
+     */
+    public function test_Add_Block_Contents_With_No_Site_Logo(): void {
+        $this->assertEquals( '', $this->logo->add_site_logo_attributes() );
+    }
```

Comment:
I think we should also have a test case for when the block content has an `<img>` tag but isn't a 'core/site-logo' block so we can test that this filter isn't interfering with `<img>`s that belong to other block types.

Reply:
the if statement would block it, similar to this one, so not sure why this would be of value

Reply:
Yeah that's what it would be testing, that that if statement is there and working as expected. If someone were to change the filter function and remove the check for the block type, the filter would be affecting all blocks but these tests wouldn't detect it.